src/examples/defaultscene.js

```diff
@@ -1,40 +1,43 @@
 // default scene loaded in src/engine/engine.js
-
 import {
   Scene,
   TorusBufferGeometry,
-  MeshNormalMaterial,
+  DirectionalLight,
   Mesh,
   Vector3,
+  MeshStandardMaterial,
+  Color,
 } from "three";
 const scene = new Scene();
 
-scene.init = () => {
-  scene.traverse(e => {
-    scene.remove(e);
-  });
+const rings = [
```

Comment:
I like the variable name change. Makes it much clearer. The first time I read `rots` I had to pause and think for...longer than I care to say.

Reply:
haha, yeah, it wasn't the most descriptive at all:)